src/website/app/App.js

```diff
@@ -119,27 +119,13 @@ class App extends Component<Props> {
     }
   }
 
-  componentDidUpdate(prevProps) {
-    if (canUseDOM && this.props.location !== prevProps.location) {
-      global.window.scrollTo(0, 0);
-    }
-  }
-
   render() {
     const { demoRoutes } = this.props;
 
     return (
       <ThemeProvider theme={siteTheme}>
         <div>
-          <Switch>
-            <Route
-              exact
-              strict
-              path="/:url*"
-              render={props => <Redirect to={`${props.location.pathname}/`} />}
```

Comment:
This will add a trailing slash to all routes that don't automatically have one. But as for why... 🤷‍♂️

Reply:
After some spelunking, the trailing slash was added because linking to a specific example route from a demo page couldn't be done like, e.g. `<a href="./example">`. But adding the trailing slash allowed for, e.g. `<a href="example">`.